endpointmanager/pkg/capabilityhandler/capabilityreceiver.go

```diff
@@ -47,12 +51,24 @@ func formatMessage(message []byte) (*endpointmanager.FHIREndpoint, error) {
 		originalURL = url
 	}
 
+	var capStat capabilityparser.CapabilityStatement
+	if msgJSON["capabilityStatement"] != nil {
+		capInt, ok := msgJSON["capabilityStatement"].(map[string]interface{})
+		if !ok {
+			return nil, fmt.Errorf("unable to cast capability statement to map[string]interface{}")
```

Comment:
It would be helpful to include the URL in this message. I am seeing this message a few times. in the console and want to know what the response looks like that is not parseable.